src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+  - web-vitals
+---
+
+We've all suffered from poor experiences where a page will "stutter" or "freeze"
+during scrolling or animations. We like to say that these experiences are not
+_smooth_. In an attempt to address these types of issues, the Chrome team has
+been working on adding more support to our lab tooling for animation detection,
+as well making steady improvements to the rendering pipeline diagnostics within
+Chromium.
+
+We'd like to share some recent progress, offer some concrete tooling guidance,
+and discuss some ideas for future animation smoothness metrics. As always, we
+would love to hear your [feedback](#feedback).
+
+This post will cover three main topics:
+
+1. A quick look at animations and animation frames.
+2. Our current thoughts on measuring overall animation smoothness.
+3. A few practical suggestions for you to leverage in lab tooling today.
+
+## What are animations?
+
+Animations bring content to life! By making content move, especially in
+response to user interactions, animations can make an experience feel more
+natural, understandable, and fun!
+
+But poorly implemented animations, or just adding too many animations, can
+degrade the experience and make it decidedly not fun at all. We've probably all
+interacted with an interface which just added too many "helpful" transition
+effects, which actually become hostile to experience when they perform poorly.
+
+You can read plenty more about [Animations](/animations/), including how to
+[create performant animations](/animations-guide/) and how to [inspect
+animations using developer
+tooling](https://developer.chrome.com/docs/devtools/css/animations/).
+
+### How do animations work?
+
+As a quick recap: the [Rendering Pipeline](/animations-overview/#pipeline)
+consists of a few, **sequential** stages:
+
+1. <strong style="font-weight:700;color:#6251A2">Style:</strong> Calculate the
+   styles that apply to the elements.
+2. <strong style="font-weight:700;color:#6251A2">Layout:</strong> Generate the
+   geometry and position for each element.
+3. <strong style="font-weight:700;color:#78A55A">Paint:</strong> Fill out the
+   pixels for each element into layers.
+4. <strong style="font-weight:700;color:#78A55A">Composite:</strong> Draw the
+   layers to the screen.
+
+While there are many ways to define animations, they all fundamentally work via
+one of:
+
+1. Adjusting <strong style="font-weight:700;color:#6251A2">layout</strong>
+   properties
+2. Adjusting <strong style="font-weight:700;color:#78A55A">paint</strong>
+   properties
+3. Adjusting <strong style="font-weight:700;color:#78A55A">composite</strong>
+   properties
+
+Because these stages are sequential, it is important to define animations in
+terms of properties further down this pipeline. The earlier the update, the
+greater the costs and the less likely it is to be smooth. (See [rendering
+performance](https://developers.google.com/web/fundamentals/performance/rendering)
+for more details.)
```

Comment:
```suggestion
Because these stages are sequential, it is important to define animations in
terms of properties that are further down the pipeline. The earlier the update happens in the process,
the greater are the costs and it's less likely to be smooth. (See [Rendering
performance](https://developers.google.com/web/fundamentals/performance/rendering)
for more details.)
```